mmocr/models/textrecog/plugins/common_plugins.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+from mmcv.cnn import PLUGIN_LAYERS
+
+
+@PLUGIN_LAYERS.register_module()
+class Maxpool2d(nn.Module):
+    """nn.Maxpool2d()
+    Args:
+        kernel_size (int | tuple): Kernel size for max pooling layer
+        stride (int | tuple): Stride for max pooling layer
+        padding (int | tuple): Padding for pooling layer
+    """
+
+    def __init__(self, kernel_size, stride, padding=0, **kwargs):
+        super(Maxpool2d, self).__init__()
+        self.model = nn.MaxPool2d(kernel_size, stride, padding)
+
+    def forward(self, x):
+        """
+        Args:
+            x (Tensor): Input feature map
+
+        Returns:
+            Tensor: The tensor after Maxpooling layer.
+        """
+        return self.model(x)
+
+
+@PLUGIN_LAYERS.register_module()
+class GCAModule(nn.Module):
+    """GCAModule in MASTER
+    Args:
+        in_channels: Channels of input tensor
+        out_channels: Channels of output tensor
```

Comment:
1. A blank line above Args
2. Pls provide the full docstring for all of the args.

Reply:
If it is just a wrapper of `conv2d` which can be covered by `ConvModule`, we don't really need it then.

Reply:
It's just a guide on how to use plugins and this is not the GCAMould in MASTER at all. I just want to show others how to use plugins.

Reply:
Since we'll be merging this PR soon, no dummy example shall be left. You might want to comment on MASTER PR with this example.

Reply:
make sense